diff -ruN lldb-14.0.5.src.orig/source/Plugins/Process/Linux/NativeRegisterContextLinux_s390x.cpp lldb-14.0.5.src/source/Plugins/Process/Linux/NativeRegisterContextLinux_s390x.cpp
--- a/lldb-14.0.5.src.orig/source/Plugins/Process/Linux/NativeRegisterContextLinux_s390x.cpp	2022-06-10 16:47:36.000000000 +0200
+++ b/lldb-14.0.5.src/source/Plugins/Process/Linux/NativeRegisterContextLinux_s390x.cpp	2022-06-20 14:28:03.551079680 +0200
@@ -18,7 +18,7 @@
 
 #include "Plugins/Process/Utility/RegisterContextLinux_s390x.h"
 
-#include <linux/uio.h>
+//#include <linux/uio.h>
 #include <sys/ptrace.h>
 
 using namespace lldb_private;
